### DISCUSSION TITLE
Is it possible to extend the webpack runtime?

### DISCUSSION THREAD

**Discussion by noahnu** [webpack/webpack#14479] [Q&A] [answered]
I'm interested in extending the webpack runtime so I can collect some metrics around number of chunks downloaded at runtime, and some other stats. Is it possible for me to provide a custom runtime to achieve this?

**Comment by alexander-akait** [comment, marked as answer]
Yes, using hooks, small example https://github.com/webpack-contrib/mini-css-extract-plugin/blob/master/src/index.js#L573
